json_yaml_files/prepare_files.py

```diff
@@ -0,0 +1,178 @@
+from bids import BIDSLayout
+from collections import defaultdict
+import re
+import json
+import sys
+import yaml
+from pathlib import Path
+
+
+class BIDStoNDAConfigGenerator:
+    def __init__(self):
+        self.path = sys.argv[1]
+        self.layout = BIDSLayout(self.path, derivatives=True)
+        self.A = 'image03'
+        self.pattern = r'^(?P<A>[^_]+)_(?P<X>[^.]+)\.(?P<Y>[^.]+)\.(?P<Z>[^.]+)\..+$'
+        # self.top_level_files = ['CHANGES',
+        #                         'README', 'dataset_description.json']
```

Comment:
```suggestion

```

Reply:
Yes. I commented it. Will delete these lines of code